main.go

```diff
@@ -2,20 +2,33 @@ package main
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
+	"github.com/bwmarrin/discordgo"
+	"golang.org/x/net/html"
 	"log"
 	"net/http"
 	"net/url"
 	"os"
 	"os/signal"
+	"strings"
 	"syscall"
+)
 
-	"github.com/bwmarrin/discordgo"
+var (
+	testOpt = flag.Bool("t", false, "テスト用です。投稿する文章をDiscordを経由せず表示します")
 )
```

Comment:
```suggestion
var testOpt = flag.Bool("t", false, "テスト用です。投稿する文章をDiscordを経由せず表示します")
```
Goに詳しくないのでふつうこういう書き方をするものか知らないのですが、カッコは不要であれば外しておいてください。